ruby/ext/google/protobuf_c/encode_decode.c

```diff
@@ -59,6 +59,30 @@ static const void *newsubmsghandlerdata(upb_handlers* h, uint32_t ofs,
   return hd;
 }
 
+typedef struct {
+  size_t ofs;      // union data slot
+  size_t case_ofs; // oneof_case field
+  uint32_t tag;    // tag number to place in data slot
```

Comment:
can we call this something else to distinguish it from a field tag number?

Does proto2 call it a discriminant or something like that?  "Case" would work too.

Also would it be more accurate to say "place in case slot"?